sdk/face/azure-ai-vision-face/src/main/java/module-info.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+module com.azure.ai.vision.face {
+    requires transitive com.azure.core;
+    exports com.azure.ai.vision.face;
+    exports com.azure.ai.vision.face.models;
+    opens com.azure.ai.vision.face.implementation.models to com.azure.core;
+    opens com.azure.ai.vision.face.models;
```

Comment:
Why open to all?

Reply:
I ran tsp-client update and then saw this file modified. Already fixed it.

Reply:
I see the problem on your test.

The failure is due to such API that has return value as generic (`List` in this case).
https://github.com/Azure/azure-sdk-for-java/blob/010eb4025d1043d8f993b0c93b0d78060bc46995/sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/FaceClient.java#L2312-L2317

And current azure-json has an issue on such https://github.com/Azure/autorest.java/issues/2629

You may choose to set `stream-style-serialization: false` in tspconfig.yaml to disable azure-json. (you need to merge the spec PR and regen the SDK).

@alzimmermsft Any recommendation on this? Is there some way for "force" BinaryData to/fromObject to use azure-json?